dockerprobe/main.go

```diff
@@ -0,0 +1,341 @@
+package main
```

Comment:
Please add an explanatory comment on top explaining why we need this and what it does 🙏

Reply:
Any additional comments within the file would be great too; but nothing specific; only for non-trivial parts

Reply:
In the top-file comment please include a sample output of the CSV structure (1-2 example lines)